catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -175,18 +204,19 @@ private MetacardImpl initializeMetacard(String id, String contentInput) {
                     .flatMap(Collection::stream)
                     .collect(Collectors.toSet());
 
-            metacard = new MetacardImpl(new MetacardTypeImpl(BasicTypes.BASIC_METACARD.getName(),
-                    BasicTypes.BASIC_METACARD,
+            metacard = new MetacardImpl(new MetacardTypeImpl(metacardType.getName(),
+                    metacardType,
                     attributes));
             for (ContentMetadataExtractor contentMetadataExtractor : contentMetadataExtractors.values()) {
                 contentMetadataExtractor.process(contentInput, metacard);
             }
         } else {
-            metacard = new MetacardImpl();
+            metacard = new MetacardImpl(metacardType);
         }
 
         metacard.setId(id);
-        metacard.setContentTypeName(MediaType.PDF.subtype());
+        metacard.setContentTypeName(MediaType.PDF.toString());
```

Comment:
Since we're planning to deprecate content-type, do you think we should start eliminating it from transformers, or continue to set it for the sake of backwards compatibility?